### ISSUE TITLE
Search [Media, Category] doesn't work

### ISSUE THREAD

**Issue by parneet-guraya** [commons-app/apps-android-commons#6043]
#### When searching in explore page. Neither `Media` nor `Category` tab show anything. Is this isn't complete yet?

https://github.com/user-attachments/assets/a46e4c62-25fb-4d57-ba57-f8fdb09532c8

#### However `Category` search does work when uploading.

 <img src="https://github.com/user-attachments/assets/79d274ac-db59-44dd-ab89-74b0cf8fb9bd" height=500> 

------------------------------------------------------------------------------
#### From this file, it seems like it's never implemented 

https://github.com/commons-app/apps-android-commons/blob/0153cbe0ed92f2ca1a6685798e5f49bd04fe967c/app/src/main/java/fr/free/nrw/commons/explore/categories/search/SearchCategoryFragment.kt

**Comment by nicolas-raoul** [comment]
I sometimes see this too.
Do you want to work on it?

Somehow sometimes it works, I got this right now, current main:
![Screenshot_20241217-222343.png](https://github.com/user-attachments/assets/2eab9ff8-5b31-4d63-855e-72bc3e9e338a)

**Comment by parneet-guraya** [comment]
> Do you want to work on it?

Yes

> Somehow sometimes it works, I got this right now, current main:

Hmm.. on `prodDebug` only `media` tab works for me. Although it still throws an error (see snackbar at the bottom). 

![Screenshot_2024-12-17-19-09-24-58_d5db3f3edc380047609fe9c266f7c566.jpg](https://github.com/user-attachments/assets/7ae84b3a-6c9f-4728-86f4-85653755c10c)

**Comment by nicolas-raoul** [comment]
Yes I agree with you: I get that snackbar too, and the categories tab seems to never work.

**Comment by prototyperspective** [comment]
Hasn't this been fixed at #5844 ?

**Comment by parneet-guraya** [comment]
Yes, this has been fixed, maybe @nicolas-raoul can close this and #5910 as well ?

**Comment by parneet-guraya** [comment]
Yes closing now. Also @nicolas-raoul this #5910 can also be closed
